src/database/stores/RethinkDbGuildStore.ts

```diff
@@ -725,12 +725,13 @@ export class RethinkDbGuildStore implements GuildStore {
         return true;
     }
 
-    public async setLogIgnores(guildId: string, userIds: string[]): Promise<boolean> {
+    public async setLogIgnores(guildId: string, userIds: string[], ignore: boolean): Promise<boolean> {
```

Comment:
you updated the signature of this method but I don't see any change in the `GuildStore` interface nor in the `log` command, do I? Why is that?

Reply:
The interface always had the `ignore:boolean` param, the impl just wasnt using it. whoops!